### PR TITLE
Remove 'primary' class from tab counter labels

### PR DESCRIPTION
**Pull request by fsiddi**
Using the primary color for each label counter makes the use of color redundant, as well as suggesting this is a call to action. Use the base grey color instead.

grey_lables

### CODE REVIEW

**Comment by delvh** [comment]
Hmm… I'm not sure about this PR:
While I can see your point, I also think that the current color is one of the most recognizable elements of the entire theme -
whenever I think of the UI, this tab appearance is the first thing that comes to mind.
So, I'm not sure if it would be a good idea to eliminate this unique characteristic.

---

Long story short:  I'll stay neutral towards this PR, I won't approve it but I am also fine if it gets merged.

**Comment by fsiddi** [comment]
This is how it looks like on the default theme.

image

**Comment by JVimes** [comment]
I like this as a fix to my suggestion. Right now the count draws more attention than the tab title. And even the tabs shouldn't draw more focus than action buttons, headings, etc.

**Comment by zeripath** [comment]
I'm gonna defer to @silverwind for their review of this

**Comment by lafriks** [comment]
> as well as suggesting this is a call to action

That's not really true, Gitea uses red for that (see notification number count badge).

I probably like better as it is now so I will also abstain

**Comment by venomgfx** [comment]
I would really like to see this merged in, for multiple reasons:

* Primary colors should be saved for call-to-actions (which are -or should be, usually limited to one per page/section).
* These labels are simple count indicators, they don't perform any action.
* The entire tab can be clicked, no need to highlight the label only.
* These indicators are often present in multiple tabs (at least in Issues, PRs, and Projects), leading to visual noise.

---

This is not totally related to this particular PR so feel free to ignore, but this hardcoded label-colors exists in other areas as well.

e.g. the repositories list in the dashboard. Labels there have the class `grey` (which is not even needed since the color is taken from context). This makes it hard to theme without getting into `!important` situations.

image

**Comment by JVimes** [comment]
> Gitea uses red for that [call to action]

I'm confident most users feel called to action by the apparent "unread count" in the tabs, regardless of the intent of the current styles.

**Comment by JVimes** [comment]
There is precedent for gray-count-in-heading in the current dashboard:
image

I think some folks are used to the way it looks and feel funny when it changes. But I think that feeling will change quickly. I really think removing a bright accent from secondary/tertiary information is better for UX and cognition.

**Review by lunny — APPROVED** [review]

**Comment by wxiaoguang** [comment]
I would also agree this change  is good. A strong reason is: other  sites also do so (say, like GitHub)

**Comment by silverwind** [comment]
Probably a good change, but I see that those grey shades need to be tweaked a bit, but can be done separately.

**Review by zeripath — APPROVED** [review]
as per @silverwind

**Review by silverwind — APPROVED** [review]

**Comment by zeripath** [comment]
please update the branch to current head

**Comment by lafriks** [comment]
Needs updating again :)

**Pull request merged**
